### ISSUE TITLE
twilio profiles:list shows inaccurate Active status

### ISSUE THREAD

**Issue by thisischrisnow** [twilio/twilio-cli#175]
### Issue Summary
~/.twilio-cli/config.json has property `activeProject: null` despite`twilio profiles:list` command showing Active `true`. This occurs after running `twilio profiles:remove 'active project'` command when two projects added and leaving a single project configured.

Command to view video apps (`twilio rtc:apps:video:view`) fails.

### Steps to Reproduce
1. Add profile for Project A with `twilio profiles:create` command
2. Add profile for Project B with `twilio profiles:create` command
3. Set Project A as active with `twilio profiles:use 'Project A'` command
4. Remove Project A with `twilio profiles:remove 'Project A'` command
5. Run command `twilio profiles:list` and see Project B shows active
6. Run command `cat ~/.twilio-cli/config.json` and see `activeProject` property shows `null`

### Technical details:
* Mac OS Mojave (10.14.6)
* twilio-cli version 4.6.0
* node.js version v10.16.3
* Command output with debug logging enabled (adding `-l debug` to the end of the command):
```
[DEBUG] Config File: /Users/User/config.json
[DEBUG] Using profile: Consumer Assessment - Dev
[DEBUG] -- BEGIN Twilio API Request --
[DEBUG] GET https://serverless.twilio.com/v1/Services
[DEBUG] User-Agent: @twilio/cli-core/4.6.0 (twilio-node/3.42.2, node.js v10.16.3, darwin 18.7.0 x64, rtc:apps:video:view)
[DEBUG] -- END Twilio API Request --
[DEBUG] response.statusCode: 200
[DEBUG] response.headers: {"date":"Thu, 23 Apr 2020 22:33:19 GMT","content-type":"application/json; charset=utf-8; charset=utf-8","content-length":"1067","connection":"close","twilio-concurrent-requests":"1","twilio-request-id":"RQ9d236c390be94cf0aa21ac8710be6610","twilio-request-duration":"0.019","access-control-allow-origin":"*","access-control-allow-headers":"Accept, Authorization, Content-Type, If-Match, If-Modified-Since, If-None-Match, If-Unmodified-Since","access-control-allow-methods":"GET, POST, DELETE, OPTIONS","access-control-expose-headers":"ETag","access-control-allow-credentials":"true","x-powered-by":"AT-5000","x-shenanigans":"none","x-home-region":"us1","x-api-domain":"serverless.twilio.com","strict-transport-security":"max-age=31536000"}
[DEBUG] -- BEGIN Twilio API Request --
[DEBUG] GET https://serverless.twilio.com/v1/Services/ZSb393adc5c07f92c6f4c2a29bab822e5a/Environments
[DEBUG] User-Agent: @twilio/cli-core/4.6.0 (twilio-node/3.42.2, node.js v10.16.3, darwin 18.7.0 x64, rtc:apps:video:view)
[DEBUG] -- END Twilio API Request --
[DEBUG] response.statusCode: 200
[DEBUG] response.headers: {"date":"Thu, 23 Apr 2020 22:33:19 GMT","content-type":"application/json; charset=utf-8; charset=utf-8","content-length":"380","connection":"close","twilio-concurrent-requests":"1","twilio-request-id":"RQ0427e646739b442f9d6fb2730cd834d8","twilio-request-duration":"0.021","access-control-allow-origin":"*","access-control-allow-headers":"Accept, Authorization, Content-Type, If-Match, If-Modified-Since, If-None-Match, If-Unmodified-Since","access-control-allow-methods":"GET, POST, DELETE, OPTIONS","access-control-expose-headers":"ETag","access-control-allow-credentials":"true","x-powered-by":"AT-5000","x-shenanigans":"none","x-home-region":"us1","x-api-domain":"serverless.twilio.com","strict-transport-security":"max-age=31536000"}
 » twilio-cli encountered an unexpected error. To report this issue, execute the command with the "-l debug" flag, then copy the output to a new issue here: https://github.com/twilio/twilio-cli/issues
[DEBUG] Cannot read property 'sid' of undefined
[DEBUG] TypeError: Cannot read property 'sid' of undefined
    at ViewCommand.getAppInfo (/Users/User/.twilio-cli/node_modules/@twilio-labs/plugin-rtc/src/helpers.js:76:64)
    at process._tickCallback (internal/process/next_tick.js:68:7)

```

**Comment by childish-sambino** [comment]
Looks unrelated to the active profile being deleted. The steps you've listed and the outputs are by-design. A `null` active project will just use the first profile that's found in the config file. You can see in the debug log `Using profile: Consumer Assessment - Dev` which would be your first profile.

As to why the command is failing, I'm not familiar with the `plugin-rtc` codebase.

@aaalaniz Can you assist here?

**Comment by aaalaniz** [comment]
Hey @thisischrisnow 

I recommend following the trouble shooting steps provided [here](https://github.com/twilio/twilio-video-app-react#troubleshooting-the-twilio-cli). Our team released an update to the plugin that should help remediate this issue.

Thanks!

**Comment by thisischrisnow** [comment]
Thank you for the response. I haven't had a chance to test again since my first attempt.
From what I remember, I was using the latest version of the code for both CLI and plugin-rtc at that time. I had downloaded both on the day I submitted the question.
I also attempted the troubleshooting steps you linked when I first encountered the issue. I would get the same output even when attempting the command `twilio rtc:apps:video:view` after trying the troubleshooting steps.

In any case, my workaround was to use internal servers rather than deploying to Twilio functions. Since this isn't the bug I thought it was, I'll close this. Thank you again.
